content/hi/docs/reference/glossary/cronjob.md

```diff
@@ -0,0 +1,19 @@
+---
+title: क्रोंजोब (CronJob)
```

Comment:
```suggestion
title: क्रोनजोब (CronJob)
```

Reply:
I think `क्रॉन्जॉब` is more similar to CronJob phenotically? WDYT @divya-mohan0209?

Reply:
Yep, phonetically it is similar. Let me amend the suggestion. Thank you @Babapool

Reply:
@divya-mohan0209 I think this suggestion will be more efifcient for pronounciation 👍